src/utils/tree_node.C

```diff
@@ -457,25 +460,30 @@ unsigned int TreeNode<N>::n_active_bins() const
 
 
 template <unsigned int N>
-const Elem* TreeNode<N>::find_element(const Point& p, const std::set<subdomain_id_type> *allowed_subdomains) const
+const Elem*
+TreeNode<N>::find_element
+(const Point& p,
+ const std::set<subdomain_id_type> *allowed_subdomains,
+ Real relative_tol) const
```

Comment:
Hell, I forgot we had "make beautify"; thanks for the reminder!  Don't even remember the last time I ran that.  Let's set up a separate PR for it the next time we don't have so many existing outstanding pull requests that it might cause conflicts in?